# 카드 트릭 관련 파라미터
card_trick:
  max_number: 10